db/collection.go

```diff
@@ -34,7 +34,7 @@ import (
 	"github.com/sourcenetwork/defradb/errors"
 	"github.com/sourcenetwork/defradb/events"
 	"github.com/sourcenetwork/defradb/lens"
-	"github.com/sourcenetwork/defradb/merkle/crdt"
+	merklecrdt "github.com/sourcenetwork/defradb/merkle/crdt"
```

Comment:
question: why alias? there no other package that conflicts with it

Reply:
the alias is automatic when the package name is different than the folder name.